### PR TITLE
Close #19785: Let WebAuthnFeature ride the trains

### LINKED ISSUE #19785: Investigate and remove feature flag for WebAuthn
We added a feature flag and this ticket is to track the removal of the flag to let it ride the trains.

Tracking feature: https://github.com/mozilla-mobile/fenix/issues/1340

### Potential Blockers
 - [ ] [#18347][18347] - [Bug] WebAuthn implementation doesn't allow retry
 - [ ] [#17688][17688] - Support Opting-in to Direct Attestations on Android
 - [x] BZ-1711541 - PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable returns false even when supported
 - [x] BZ-1689612 - WebAuthn does not specify the authenticator type during attestation

[18347]: https://github.com/mozilla-mobile/fenix/issues/18347
[17688]: https://github.com/mozilla-mobile/fenix/issues/17688

### PR DESCRIPTION
**Pull request by jonalmeida**
We have the main blockers for this feature now fixed, so let's allow it to go into beta/release.

### CODE REVIEW

**Comment by jonalmeida** [comment]
Intermittent test failures that have already been filed.

**Review by csadilek — APPROVED** [review]
🚂

File: app/src/main/java/org/mozilla/fenix/FeatureFlags.kt [hunk 1]
```diff
@@ -27,7 +27,7 @@ object FeatureFlags {
     /**
      * Enables WebAuthn support.
      */
-    val webAuthFeature = Config.channel.isNightlyOrDebug
+    val webAuthFeature = true
```
Comment (csadilek) [inline app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:30]: There's a lint error. It wants this to be a `const`:
MayBeConst - [webAuthFeature] at /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:30:9

Comment (jonalmeida) [reply app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:30]: Ah heck, this is why I shouldn't try to edit a patch in the Github UI. 🙃

**Pull request merged**